.github/ISSUE_TEMPLATE/report-an-issue.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Report an issue
+about: Help us improve our service by reporting issues you are experiencing.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the issue**
+A clear and concise description of what is the issue.
+
+**Expected result vs actual result**
```

Comment:
The second part here (actual result) seems redundant to ``What error did you get?``.

I would also probably put this after the reproduction steps and maybe have:

```
**Steps to Reproduce**
<!-- Steps to reproduce the behavior (if applicable) -->

**Actual Result**

**Expected Result**

```

And just drop the ``What error did you get?`` question.